{{cookiecutter.project_slug}}/local.yml

```diff
@@ -9,6 +9,10 @@ services:
     build:
       context: .
       dockerfile: ./compose/local/django/Dockerfile
+      {%- if cookiecutter.use_watchman == 'y' %}
+      args:
+        WATCHMAN_VERSION: v2021.03.01.00
```

Comment:
I feel like this isn't necessary to be placed here and instead just be placed in the docker file itself.

Reply:
Ah yes, I had missed that ARG can take a default value. Makes sense to put it in the Dockerfile